resources/sass/pages/_cart.scss

```diff
@@ -0,0 +1,212 @@
+.flex {
+    @include flex;
+}
+
+.flex-row {
+    @include flex-row;
+}
+
+.flex-column {
+    @include flex-column;
+}
+
+.flex-row-reverse {
+    @include flex;
+    flex: 1;
+    flex-direction: row-reverse;
+}
+
+.flex-column-reverse {
+    @include flex;
+    flex: 1;
+    flex-direction: column-reverse;
+}
+
+main.cart {
+    @extend .flex-column;
+    background: #eeeeee;
+    padding: 33px 0px;
+    max-width: 1140px;
+    margin: 0 auto;
+    position: relative;
+    .cart-header {
+        @extend .flex-column;
+        background: $color_black;
+        padding: 10px;
+        &__text {
+            color: $color_white;
+        }
+    }
+}
+
+.cart-item {
+    @extend .flex-row;
+    margin-top: 10px;
+    background: $color_white;
+    &__logo {
+        width: 20%;
+        height: 20%;
+    }
+}
+
+.cart-item-detail {
+    @extend .flex-column;
+    &__heading {
+        font-size: 1.5rem;
+        padding: 15px;
+    }
+    &__info {
+        @extend .flex-row;
+        font-size: 1rem;
+        align-items: center;
+        &--count {
+            @extend .flex-row;
+        }
+        &--remove,
+        &--add {
+            margin-right: 25px;
+            margin-left: 25px;
+            height: 25px;
+            width: 25px;
+            background: $color_red;
+            text-align: center;
+            color: $color_white;
+            border-radius: 100%;
+            display: inline-flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            font-size: 1rem;
+            font-weight: bold;
+            text-decoration: none;
+            font-family: sans-serif;
+        }
+        &--total {
+            margin-right: 50px;
+        }
+    }
+}
+
+.cart__content__price {
+    margin-top: 16px;
+    width: 100%;
+    padding: 12px;
+    display: flex;
+    align-items: center;
+    background: $color_white;
+}
+
+.button-small-checkout {
+    border: $color_red;
+    font-size: 100%;
+    padding: 10px;
+    background: $color_red;
+    color: $color_white;
+    width: 100%;
+    text-align: left;
+}
+
+.button-price {
+    float: right;
+}
+
+.checkout-box {
+    margin-top: 16px;
+    width: 100%;
+    padding: 12px;
+    background: $color_white;
+    .checkout-text {
+        text-align: center;
+    }
+}
+
+.cart-modal {
+    display: none;
+    position: fixed;
+    z-index: 1;
+    left: 31%;
+    top: 0;
+    width: 100%;
+    height: 87%;
+    top: 12%;
+    &-content {
+        position: absolute;
+        top: 0;
+        left: 0%;
+        bottom: 0;
+        background: #eeeeee;
+        margin-left: 35%;
+        margin-right: 35%;
+        border: 1px solid $color_black;
+    }
+    &-header {
```

Comment:
This is not correct BEM naming convention

Reply:
I already fixed it.